src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs

```diff
@@ -34,8 +34,8 @@ public async Task<IActionResult> GetCriminalCompliancePage(Guid applicationId, s
             var username = _contextAccessor.HttpContext.User.UserDisplayName();
             var viewModel = await _orchestrator.GetCriminalComplianceCheckViewModel(new GetCriminalComplianceCheckRequest(applicationId, gatewayPageId, username));
 
-            viewModel.PageTitle = CriminalCompliancePageConfiguration.Titles[gatewayPageId];
-
+            //viewModel.PageTitle = CriminalCompliancePageConfiguration.Titles[gatewayPageId];
```

Comment:
If this assertion isn't needed can it be deleted rather than commented out?

Reply:
I don't think that should have been commented out

Reply:
We don't need PageTitle. It's constructed as everywhere else using the heading => ViewBag.Title = $"Gateway - {Model.Heading}";

Reply:
Ok, for criminal compliance pages there are nearly 20 different versions with different titles based on the question ID, so the Model.Heading will need to be populated from that.

Reply:
Some headings were not populated as Titles as well. That is why I asked for 
“You will also need to populate the rest of the headings in the following file (// TODO Not Assigned)”
 \das-admin-service\src\SFA.DAS.AssessorService.ApplyTypes\Roatp\CriminalCompliancePageConfiguration.cs

Reply:
comment deleted